Makefile

```diff
@@ -14,7 +14,7 @@ MAKEFLAGS += --no-builtin-rules
 
 # todo: leave the default recipe prefix for now
 ifeq ($(origin .RECIPEPREFIX), undefined)
-  $(error This Make does not support .RECIPEPREFIX. Please use GNU Make 4.0 or later)
+$(error This Make does not support .RECIPEPREFIX. Please use GNU Make 4.0 or later)
```

Comment:
intentional?

Reply:
Yes, completely deliberate.

This PR ended up with indenting everything with tabs, except for this single line. This one can't be indented with tabs, because that would cause syntax error

```
Makefile:17: *** commands commence before first target.  Stop.
```

The only way to achieve consistent indentation in the whole file is to not indent line 17.